### ISSUE TITLE
Venn diagram generator

### ISSUE THREAD

**Issue by orwant** [google/google-visualization-issues#1726]
```
(No text was entered with this change)
```

Original issue reported on code.google.com by `None` on 1970-01-01T00:00:00.00Z

**Comment by nguyen-brian** [comment]
👍 would like to see this chart type ported over from the [old API](https://developers.google.com/chart/image/docs/gallery/venn_charts).  Specifically in our app, it would be perfect for a visualization of "people opted into notification categories A, B, and C, or any combination thereof".

**Comment by rob837428394** [comment]
Yes please. It would be great to be able to visualise data with Venn charts. Thank you
